homeassistant/components/smartthings/cover.py

```diff
@@ -70,6 +70,8 @@ def get_capabilities(capabilities: Sequence[str]) -> Sequence[str] | None:
 class SmartThingsCover(SmartThingsEntity, CoverEntity):
     """Define a SmartThings cover."""
 
+    _attr_supported_features: int
```

Comment:
The base class allows `None` values, but this causes with `mypy`.
Since it gets defined in `__init__`, we can safely override it here.